Source/Alamofire.swift

```diff
@@ -91,279 +95,292 @@ extension URLRequest {
     }
 }
 
-// MARK: - Request Methods
-
-/**
-    Creates a request using the shared manager instance for the specified method, URL string, parameters, and
-    parameter encoding.
-
-    - parameter method:     The HTTP method.
-    - parameter URLString:  The URL string.
-    - parameter parameters: The parameters. `nil` by default.
-    - parameter encoding:   The parameter encoding. `.URL` by default.
-    - parameter headers:    The HTTP headers. `nil` by default.
-
-    - returns: The created request.
-*/
+// MARK: - Data Request
+
+/// Creates a data `Request` using the default session manager instance to retrieve the contents of the specified
+/// `method`, `urlString`, `parameters`, `encoding` and `headers`.
+///
+/// - parameter urlString:  The URL string.
+/// - parameter method:     The HTTP method.
+/// - parameter parameters: The parameters. `nil` by default.
+/// - parameter encoding:   The parameter encoding. `.URL` by default.
+/// - parameter headers:    The HTTP headers. `nil` by default.
+///
+/// - returns: The created data `Request`.
 @discardableResult
 public func request(
-    _ method: Method,
-    _ URLString: URLStringConvertible,
+    _ urlString: URLStringConvertible,
+    withMethod method: HTTPMethod,
     parameters: [String: AnyObject]? = nil,
     encoding: ParameterEncoding = .url,
     headers: [String: String]? = nil)
     -> Request
 {
-    return Manager.sharedInstance.request(
-        method,
-        URLString,
+    return SessionManager.default.request(
+        urlString,
+        withMethod: method,
         parameters: parameters,
         encoding: encoding,
         headers: headers
     )
 }
 
-/**
-    Creates a request using the shared manager instance for the specified URL request.
-
-    If `startRequestsImmediately` is `true`, the request will have `resume()` called before being returned.
-
-    - parameter URLRequest: The URL request
-
-    - returns: The created request.
-*/
+/// Creates a data `Request` using the default session manager instance to retrieve the contents of the
```

Comment:
We may want to be close to Apple's verbiage here: `retrieves the contents of a URL based on the specified URL request object`. Technically, you don't retrieve the contents of the `URLRequest`. But that's minor.